integration-tests/cypress/e2e/project/deck/deck.cy.ts

```diff
@@ -0,0 +1,300 @@
+        studioProjectId = payload.project.id;
+
+        new Project().visit(studioProjectId);
+        cy.getByTestId('new-model').click();
+        cy.getByTestId('name-input').type('Deck View');
+        cy.getByTestId('stereotype').click();
+        cy.get('li').filter(':contains("Deck View")').click();
+        cy.getByTestId('create-document').click();
+      }
+    });
+  });
+  after(() => {
+    // We delete the created studio once all tests have been executed
+    cy.deleteProject(studioProjectId);
+  });
+
+  context('We first verify the creation of an empty deck representation', () => {
+    it('We create the Deck Representation', () => {
+      createTaskProjectAndDeckRepresentation('Project Dev', 'New Daily Representation');
+      new Explorer().getSelectedTreeItems().contains('New Daily Representation').should('exist');
+      new Deck().getDeckRepresentation().should('exist');
+
+      deleteCurrentProject();
+    });
+  });
+
+  context('We new verify the Deck Creation, Deletion and renaming', () => {
+    let taskProjectId: string;
+    before(() => {
+      createTaskProjectAndDeckRepresentation('Daily Project Dev', 'New Daily Representation').then(() => {
+        cy.url().then(($url) => {
+          const prefix = Cypress.config().baseUrl + '/projects/';
+          taskProjectId = $url.substring(prefix.length, $url.indexOf('/', prefix.length + 1));
+        });
+      });
+    });
+
+    beforeEach(() => {
+      openDeckRepresentation(taskProjectId, 'Daily Project Dev', 'New Daily Representation');
+    });
+
+    after(() => {
+      cy.deleteProject(taskProjectId);
+    });
+
+    it('We verify the Deck Representation on the Daily project', () => {
+      new Explorer().getSelectedTreeItems().contains('New Daily Representation').should('exist');
+
+      //We verify that all lanes are present.
+      const days: string[] = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
+      days.forEach((day) => {
+        new Deck().getLane(day).should('exist');
+      });
+
+      //We verify that all cards are present in the expected lane.
+      cy.getByTestId('lane-Monday').findByTestId('card-Idea').should('exist');
+      cy.getByTestId('lane-Monday').findByTestId('card-Specification').should('exist');
+      cy.getByTestId('lane-Tuesday').findByTestId('card-Development').should('exist');
+      cy.getByTestId('lane-Friday').findByTestId('card-Release').should('exist');
+    });
+
+    it('We rename the Deck Representation on the Daily project', () => {
+      new Explorer().getTreeItemByLabel('New Daily Representation').should('exist');
+      new Deck().getDeckRepresentation().should('exist');
+      new Explorer().rename('New Daily Representation', 'renamed_New Daily Representation');
+      new Explorer().getTreeItemByLabel('New Daily Representation').should('not.exist');
+      new Explorer().getTreeItemByLabel('renamed_New Daily Representation').should('exist');
+
```

Comment:
No need to instantiate a `Explorer` each time, prefer `const explorer = new Explorer();`